a11y/Dockerfile

```
@@ -0,0 +1,39 @@
+FROM node:15.0.1
+
+LABEL maintainer="daniel.walford@ons.gov.uk"
+
+# Explicitly declare npm user is root (docker runs as root anyway) allows global package installation for npm, needed for @axe-core/cli
+RUN npm --global config set user root
+
+# Update apt then install wget to set up the Personal Package Archive (PPA) and unzip to install the Chromedriver
+RUN apt update \
+    && apt install -y wget unzip
```

Running apt can bloat the container resulting in a bigger image and slower pull. 

A couple of best practices that can prevent this are:

```suggestion
RUN apt update \
    && apt install -y --no-install-recommends wget unzip \
    && rm -rf /var/lib/apt/lists/*
```